src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/DocumentGeneration.java

```diff
@@ -19,6 +19,8 @@ public class DocumentGeneration {
     private YesNo generateNotice;
     @JsonProperty("bodyContent")
     private String bodyContent;
+    @JsonProperty("bodyContentCorrection")
```

Comment:
Is it worth prefixing bodyContent with correction here (if there's any further correction fields to be added here, if not ignore me).

Reply:
Will consider renaming on a future branch if necessary. We have agreed that this syntax is fine here for now.